kafka-ui-api/src/test/java/com/provectus/kafka/ui/AbstractBaseTest.java

```diff
@@ -20,41 +18,36 @@
 @SpringBootTest
 @ActiveProfiles("test")
 public abstract class AbstractBaseTest {
+    private static final String CONFLUENT_PLATFORM_VERSION = "5.2.1";
+
+    public static final KafkaContainer kafka = new KafkaContainer(DockerImageName.parse("confluentinc/cp-kafka").withTag(CONFLUENT_PLATFORM_VERSION))
+            .withNetwork(Network.SHARED);
+
+    public static final SchemaRegistryContainer schemaRegistry = new SchemaRegistryContainer(CONFLUENT_PLATFORM_VERSION)
+            .withKafka(kafka)
+            .dependsOn(kafka);
+
+    public static final KafkaConnectContainer kafkaConnect = new KafkaConnectContainer(CONFLUENT_PLATFORM_VERSION)
+            .withKafka(kafka)
+            .waitingFor(Wait.forHttp("/"))
+            .dependsOn(kafka)
+            .dependsOn(schemaRegistry)
+            .withStartupTimeout(Duration.ofMinutes(15));
+
```

Comment:
Don't you think 15 is the too long timeout?

Reply:
It is, changed it to 10.
5 minutes is too short in my local dev environment with running `docker-compose` with dev clusters, but should be okay for CI.
Should I reduce it to 5 minutes?

Reply:
Yes, I think 5 minutes timeout is quite enough

Reply:
It's interesting why is it starting up so long? Haven't you researched this issue?

Reply:
Reduced to 5 minutes.
Most of the startup time is discovering and regsitering all the connector plugins.
Actually in perfect environment 2 minutes should be enough, but I prefer to leave 5 just in case